src/Microsoft.Health.Fhir.Liquid.Converter/Json/JsonDataParser.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Liquid.Converter.Exceptions;
+using Microsoft.Health.Fhir.Liquid.Converter.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Json
+{
+    public class JsonDataParser
```

Comment:
>JsonDataParser [](start = 17, length = 14)

can we have a shared interface?

Reply:
Sure. According to offline discussion we'll do several refinements for code structure. I'll do it in another PR.